lib/modules/manager/bazelisk/extract.ts

```diff
@@ -0,0 +1,12 @@
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile {
+  const dep: PackageDependency = {
+    depName: 'bazel',
+    currentValue: content.trim().split('\n', 2)[0].trim(),
```

Comment:
```suggestion
    currentValue: content.split('\n', 2)[0].trim(),
```
i think the first trim can be removed

Reply:
I think that means we can't depend on `[0]` being the first non-empty line?

Reply:
i think trim will only remove spaces, no newlines?

Reply:
If it's the first non empty line you need then better to split, map trim, filter Boolean, then take the first.

Reply:
> i think trim will only remove spaces, no newlines?

Trim essentially removes the regex `\w`, so any whitespace including newlines, tab etc.

I'll just remove it for now like suggested. If these files can be more complicated (such as comments above the version number) it can be a followup PR.